.tool-versions

```diff
@@ -1,3 +1,3 @@
-erlang 23.0.3
-elixir 1.10.4-otp-23
-rust 1.46.0
+elixir 1.11.2
```

Comment:
the latest version of Elixir is `1.11.3`

Reply:
Yeah but we don’t use 1.11.3 anywhere else.